### PR TITLE
✨ Add possibility to remove hook

### PR DESCRIPTION
**Pull request by thibmaek**
## Description

Introduces the new command `--remove|-r` to unlink a previously set hook. Avoids having to dive into the .git dir in a terminal and remove it there. I love Gitmoji, but sometimes needed to remove it for projects where other people didn't want to use it.

It uses `fs#unlink` which is an async method that takes the path and a completion callback. Very much like `fs#writeFile`.

## Tests

- [x] All tests passed.

### CODE REVIEW

File: src/gitmoji.js [hunk 1]
```diff
@@ -75,6 +75,24 @@ class GitmojiCli {
 		};
 	}
 
+	remove() {
+		const hookFile = 'prepare-commit-msg';
```
Comment (carloscuesta) [inline src/gitmoji.js]: Can you delete `hookFile` and `path` constants from `remove()` and init `methods()` and create a global `commitHookPath` constant outside the class. Code duplication is dangerous! ⚠️

File: src/gitmoji.js [hunk 2]
```diff
@@ -75,6 +75,24 @@ class GitmojiCli {
...
+		const path = `${process.env.PWD}/.git/hooks/${hookFile}`;
```
Comment (carloscuesta) [inline src/gitmoji.js]: Same as hookFile ☝️

**Comment by thibmaek** [comment]
Sure, think I fixed it now.

**Comment by carloscuesta** [comment]
Great! Take a look at the build @thibmaek the tests are failing 🚨

**Comment by thibmaek** [comment]
Woops, you're right! Totally forgot to update the test property but should be fine now. Also added a path check for `gitmoji#remove`

**Pull request merged**